### PR TITLE
Add conversion for power function and some tests

### PR DESCRIPTION
**Pull request by Thorsten71**

### CODE REVIEW

File: src/specification.js [hunk 1]
```diff
@@ -712,10 +719,16 @@ var Specification = extend(null, /** @lends Specification# */ {
...
             var unitFamily = sourceUnit && sourceUnit.unitFamily;
 
+            var hasPower = _.isNumber(conversion.power);
             var hasFactor = _.isNumber(conversion.factor);
             var hasOffset = _.isNumber(conversion.offset);
-            var autoConvert = !hasFactor && !hasOffset;
+            var autoConvert = !hasFactor && !hasOffset && !hasPower;
 
+            if (hasPower) {
+                if (rawValue !== 0) {
```
Comment (danielwippermann) [inline src/specification.js]: Please remove that `if` and adapt tests. The result of `Math.pow(0, 0)` is 1 and not 0.

File: src/specification.js [hunk 2]
```diff
@@ -650,6 +650,12 @@ var Specification = extend(null, /** @lends Specification# */ {
             if (_.isNumber(conversion.factor)) {
                 invertedConversion.factor = 1 / conversion.factor;
             }
+            if (_.isNumber(conversion.power)) {
+                invertedConversion.power = conversion.power;
```
Comment (danielwippermann) [inline src/specification.js]: Please put this in an `else` branch of the `if` below.

File: test/specs/specification.spec.js [hunk 3]
```diff
@@ -1035,7 +1079,7 @@ describe('Specification', function() {
             expect(invertedConversions[1]).property('offset').equal(-10);
         });
 
-        it('should work correctly with multiple conversions using a manual °C -> °F formula', function() {
+        it('should work correctly with multiple conversions using a manual °C -> °F formular', function() {
```
Comment (danielwippermann) [inline test/specs/specification.spec.js]: Typo? :)

File: src/specification.js (hunk 1 above)
Comment (aldoinireland) [inline src/specification.js]: Dear friend! 

Just take a look at that cool  stuff I just have  found, it's so fantastic! Here  it is  <http://success.sisterscouture.com/aerowa>

Warm regards, alan

File: test/specs/specification.spec.js (hunk 3 above)
Comment (aldoinireland) [inline test/specs/specification.spec.js]: Hey! 

I've got a nice surprise for you, I think you'll love it! Just take  a look <http://period.davidwoodruff.org/aebcuvl>

My Best, alan

**Comment by aldoinireland** [comment]
Hey, 

I've created a couple  of web-sites and  wanted to ask you  some questions abou  them, please answer them here <http://put.slidersfan.com/aeirvh>. Thank you!


Hugs, alan

**Pull request merged**